tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -146,6 +146,37 @@ def test_positive_check_available_networks(self, azurermclient, module_azurerm_c
         portal_nws = azurermclient.list_network()
         assert len(portal_nws) == len(cr_nws['results'])
 
+    @pytest.mark.stubbed
+    @pytest.mark.tier2
+    def test_gov_cloud_regions_from_azure_compute_resources(self):
+        """Check Azure Gov Cloud for US Cloud region options when creating a compute resource
+
+        :id: 65e03f7e-a2c7-4541-b92b-38d80ab48175
+
+        :CaseImportance: Medium
+
+        :CaseLevel: Acceptance
+
+        :CaseAutomation: NotAutomated
+
+        :steps:
+            1. Create Azure Compute Resource
+            2. Select US Government Cloud option
+            3. Input Client ID, tenant ID, Secret, and Sub Id
+            4. Verify you can select US regions
+            5. Test connection and create compute resource
+
+        :expectedresults: All regions below should be visible
+            1. US DoD Central
+            2. US DoD East
+            3. US Gov Arizona
+            4. US Gov Iowa
+            5. US Gov Texas
+            6. US Gov Virginia
+
+        :BZ: 1829107
```

Comment:
This metadata should include `customer_scenario: true`

Reply:
as @devendra104 pointed out the testimony token doesn't actually have an underscore, always check against testimony.yaml or run pre-commit.